cmake/compiler/gcc/compiler_flags.cmake

```diff
@@ -21,7 +21,7 @@ endif()
 set_compiler_property(PROPERTY optimization_speed -O2)
 set_compiler_property(PROPERTY optimization_size  -Os)
 set_compiler_property(PROPERTY optimization_size_aggressive -Oz)
-set_compiler_property(PROPERTY optimization_fast -Ofast)
+set_compiler_property(PROPERTY optimization_fast -O3)
```

Comment:
If we choose to make this change, I'll add a context comment here so it doesn't accidentally get changed back in the future.